src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private double side;
+
+    public Square(String color, double side) {
+        super(color);
+        this.side = side;
+    }
+
+    @Override
+    public String toString() {
+        return "Figure: square, area: " + Math.pow(side, 2) + " sq.units, side: " + side
+                + " units, color: " + new ColorSupplier().getRandomColor();
```

Comment:
Think about which variables should be local in the method and which should be class-level